vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -101,12 +105,22 @@ public VariantExporter(boolean excludeAnnotations) {
             }
         } catch (AnnotationMetadataNotFoundException e) {
             logger.warn("Annotation metadata not found, no variants will be exported for the region: " + region, e);
+        } catch (Exception e) {
+            logger.error("Could not export region '" + region + "'. ", e);
+            throw e;
         }
 
-
         return variantsToExport;
     }
 
+    private int castSafely(Long inputNumber) {
```

Comment:
I would find the code more obvious this way, and it only tries the cast when it can actually succeed:

```java
if (inputNumber > Integer.MAX_VALUE) {
    throw new RuntimeException("Tried to cast a long into an int, but it didn't fit");
} else {
    return inputNumber.intValue();
}
```

Reply:
I don't really like to assume that's the only reason the cast would fail. in this class, the number will be positive always, but shouldn't a "castSafely" function check also `if (inputNumber < Integer.MIN_VALUE)`? 

It seems more safe to me to use the current condition: it's ok if converting back yields the initial value. I don't mind using `Integer.longValue()` instead of a plain cast if that's more intuitive